src/localization/fi.ts

```diff
@@ -58,6 +58,7 @@ export const messages = {
 
   'main.mentorsTitleAndSearchButton': 'Hae',
   'main.mentorsTitleAndSearchButtonFiltersActive': 'Filtterit',
+  'main.searchMentor.hideInactiveMentors': 'Piilota ei-aktiiviset mentorit',
```

Comment:
Please ask Eppu about that these translations are ok! =)

Reply:
The Finnish translate I toke it from Figma

Reply:
Sure! But it is not probably made by Eppu. At this point we can just verify that it is ok =)! It might be already fine.

Reply:
He told me that he gonna decide tomorrow in the sprint